apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/GoogleCloudConsoleFormatter.cs

```diff
@@ -177,6 +185,35 @@ private static void WriteKeyValuePairs(Utf8JsonWriter writer, IEnumerable<KeyVal
         writer.WriteEndObject();
     }
 
+    private void MaybeWriteTraceInformation(Utf8JsonWriter writer)
+    {
+        if (string.IsNullOrWhiteSpace(_options.TraceGoogleCloudProjectId))
+        {
+            return;
+        }
+
+        var activity = Activity.Current;
+        if (activity == null)
+        {
+            return;
+        }
+
+        var traceId = activity.TraceId;
+
+        // The `Equals` method of the `ActivityTraceId` struct uses the _hexString field to compare two ActivityTraceId instances.
+        // The ActivityTraceId struct has a default value with an uninitialized _hexString field.
+        // So, `Equals` method doesn't work as expected. String comparision is unfortunate but currently the only available workaround.
+        if (traceId.ToString() == default(ActivityTraceId).ToString())
```

Comment:
Given that we're using the string representation just below, let's just call `ToString()` once and put it in a local variable.

Additionally, `default(ActivityTraceId).ToString()` presumably returns the same thing each time - so let's keep that in a static readonly variable, rather than calling `ToString()` each time.